src/calculateRentalCost.js

```diff
@@ -6,5 +1,22 @@
+const LONG_TERM = 3;
+const BASEPRISE = 40;
+const SELL = 20;
```

Comment:
The variable `SELL` is not clearly named. It would be better to rename it to something like `DISCOUNT_FOR_LONG_TERM` to indicate its purpose more clearly.